README.md

```diff
@@ -49,20 +50,23 @@ Upgrade pipx with `brew update && brew upgrade pipx`.
 sudo apt update
 sudo apt install pipx
 pipx ensurepath
+sudo pipx ensurepath --global # optional to allow pipx actions with --global argument
 ```
 
 - Fedora:
 
 ```
 sudo dnf install pipx
 pipx ensurepath
+sudo pipx ensurepath --global # optional to allow pipx actions with --global argument
 ```
 
 - Using `pip` on other distributions:
 
 ```
 python3 -m pip install --user pipx
 python3 -m pipx ensurepath
+sudo pipx ensurepath --global # optional to allow pipx actions with --global argument
```

Comment:
```suggestion
sudo pipx ensurepath --global # optional on Linux systems to allow pipx actions with --global argument
```

Reply:
This is already under the `Linux` section, so I think this is redundant.